graphql-java-kickstart/src/main/java/graphql/kickstart/execution/GraphQLInvoker.java

```diff
@@ -27,25 +28,36 @@
   }
 
   public GraphQLQueryResult query(GraphQLInvocationInput invocationInput) {
+    return queryAsync(invocationInput).join();
```

Comment:
I preserved the original signatures for binary compatibility.

This one is also still used by `AbstractGraphQLHttpServlet.executeQuery()` for JMX.